src/ast/mod.rs

```diff
@@ -613,13 +613,15 @@ pub enum Expr {
     /// `[NOT] LIKE <pattern> [ESCAPE <escape_character>]`
     Like {
         negated: bool,
+        any: bool,
         expr: Box<Expr>,
         pattern: Box<Expr>,
         escape_char: Option<String>,
     },
     /// `ILIKE` (case-insensitive `LIKE`)
     ILike {
         negated: bool,
+        any: bool,
```

Comment:
Could we add a link to the snowflake doc where the syntax is mentioned? It would help with context for someone that comes across it in the codebase

Reply:
Addressed in the latest commit